stripe/src/test/java/com/stripe/android/StripeTest.java

```diff
@@ -849,23 +849,36 @@ public void retrieveSourceAsync_withValidData_passesIntegrationTest() throws Str
         );
     }
 
-    @Test
-    public void retrieveSourceSynchronous_withValidData_passesIntegrationTest()
-            throws StripeException {
-        final Source source = createSource();
-
-        final String sourceId = source.getId();
-        final String clientSecret = source.getClientSecret();
-        assertNotNull(sourceId);
-        assertNotNull(clientSecret);
-
-        final Source retrievedSource = createStripe()
-                .retrieveSourceSynchronous(sourceId, clientSecret);
-
-        // We aren't actually updating the source on the server, so the two sources should
-        // be identical.
-        assertEquals(source, retrievedSource);
-    }
+  @Test
+  public void retrieveSourceSynchronous_withValidData_passesIntegrationTest()
```

Comment:
this test can live in `StripeApiRepositoryTest` - look at the other tests in that class for an example of how to verify analytics events

Reply:
should I move the old one here or just leave it?